nextflow_schema.json

```diff
@@ -93,6 +93,126 @@
             },
             "fa_icon": "fas fa-file-signature"
         },
+        "annotation_bakta": {
+            "title": "Annotation: BAKTA",
+            "type": "object",
+            "description": "These parameters influence the annotation algorithm of Bacteria used by BAKTA.",
+            "default": "",
+            "properties": {
+                "annotation_bakta_mincontiglen": {
+                    "type": "integer",
+                    "default": 1,
+                    "description": "Specify the minimum contig size.",
+                    "help_text": "Specifies the minimum contig size that would be annotated by BAKTA.\nIf run with '--compliant', the minimum contig length must be set to 200. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--min-contig-length`",
+                    "minimum": 1,
+                    "maximum": 200,
```

Comment:
only if `--compliant` is given no? I think `minimum`, which we don't by default - 
```suggestion
```